src/common/components/table/table-examples/MempoolTable.tsx

```diff
@@ -0,0 +1,391 @@
+      <Flex alignItems="center" justifyContent="flex-end" w="full">
+        <DefaultTableColumnHeader header={header}>Fee</DefaultTableColumnHeader>
+      </Flex>
+    ),
+    accessorKey: TxTableColumns.Fee,
+    cell: info => (
+      <Flex alignItems="center" justifyContent="flex-end" w="full">
+        {FeeCellRenderer(info.getValue() as string)}
+      </Flex>
+    ),
+    enableSorting: true,
+  },
+  {
+    id: TxTableColumns.BlockTime,
+    header: ({ header }: { header: Header<MempoolTableData, unknown> }) => (
+      <Flex alignItems="center" justifyContent="flex-end" w="full">
+        <DefaultTableColumnHeader header={header}>Timestamp</DefaultTableColumnHeader>
+      </Flex>
+    ),
+    accessorKey: TxTableColumns.BlockTime,
+    cell: info => (
+      <Flex alignItems="center" justifyContent="flex-end" w="full">
+        {TimeStampCellRenderer(
+          formatTimestampToRelativeTime(info.getValue() as number),
+          formatTimestamp(info.getValue() as number, 'HH:mm:ss', true)
+        )}
+      </Flex>
+    ),
+    enableSorting: false,
+    meta: {
+      tooltip: 'Timestamps are shown in your local timezone',
+    },
+  },
+];
+
+const TX_TABLE_PAGE_SIZE = 20;
+
+export const UpdateTableBannerRow = ({ onClick }: { onClick: () => void }) => {
+  const numColumns = Object.keys(TxTableColumns).length;
+
+  return (
+    <ChakraTable.Row
+      bg="transparent"
+      css={{
+        '& > td:first-of-type': {
+          borderTopLeftRadius: 'redesign.md',
+          borderBottomLeftRadius: 'redesign.md',
+        },
+        '& > td:last-of-type': {
+          borderTopRightRadius: 'redesign.md',
+          borderBottomRightRadius: 'redesign.md',
+        },
+      }}
+      onClick={onClick}
+      cursor="pointer"
+      className="group"
+    >
+      <ChakraTable.Cell colSpan={numColumns} py={2} px={1}>
+        <Flex
+          alignItems="center"
+          justifyContent={{ base: 'flex-start', md: 'center' }}
+          gap={1.5}
+          boxShadow="0px 4px 4px 0px rgba(252, 100, 50, 0.25), 0px 4px 4px 0px rgba(255, 85, 18, 0.25)"
+          border="1px dashed var(--stacks-colors-accent-stacks-500)"
+          borderRadius="redesign.lg"
+          h={12}
+          px={4}
+        >
+          <Box display="inline-flex">
+            <Text fontSize="sm" fontWeight="medium" color="textSecondary">
+              New transactions have come in.
+            </Text>
+            &nbsp;
+            <Text
+              fontSize="sm"
+              fontWeight="medium"
+              color="textSecondary"
+              _groupHover={{ color: 'textPrimary' }}
+            >
+              Update list
+            </Text>
+          </Box>
+          <Icon h={3.5} w={3.5} color="iconTertiary" _groupHover={{ color: 'iconSecondary' }}>
+            <ArrowsClockwise />
+          </Icon>
+        </Flex>
+      </ChakraTable.Cell>
+    </ChakraTable.Row>
+  );
+};
+
+function getToAddress(tx: MempoolTransaction): string {
```

Comment:
can we take these 2 utility functions put them in a util function and reuse them in both table components?